.github/workflows/deploy-snapshot.yml

```diff
@@ -41,6 +41,11 @@ jobs:
           twine upload dist/*
         shell: bash
 
+        - name: Upload coverage to Codecov
```

Comment:
i think the indentation here is 1 off, which i believe causes problems for yaml files.